.github/workflows/codeql.yml

```diff
@@ -14,10 +12,6 @@ jobs:
       # required for all workflows
       security-events: write
 
-      # only required for workflows in private repositories
-      actions: read
-      contents: read
```

Comment:
So the removal is based on that comment right?
And here private in the sense visibility or fork of a public project are considered private?